static/navbar/index.html

```diff
@@ -4,58 +4,68 @@
             <a href="https://stjude.cloud">St. Jude Cloud</a>
           </span>
           <span class="site-title mr-auto">
-            <span>Portal Title</span>
+            <span>App Title</span>
           </span>
 
           <ul class="nav justify-content-center global-icons">
-            <li class="nav-item active" id="icon-platform" title="About St. Jude Cloud">
-              <a class="nav-link" href="https://stjude.github.io/sjcloud-docs/index.html">
-                <i class="fa fa-info-circle"></i>
-              </a>
-            </li>
+            
             <li class="nav-item nav-item-apps dropdown">
               <a class="nav-link dropdown-toggle" href="/" id="nav-item-icon-apps" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="St. Jude Cloud Apps">
-                <i class="fa fa-th"></i>
+                <i class="fas fa-bars"></i>
               </a>
-              <div class="dropdown-menu" aria-labelledby="nav-item-icon-apps">
-                <a class="dropdown-item" href="http://cstn-ui.azurewebsites.net">CSTN</a>
-                <a class="dropdown-item" href="http://nbtp-dev.azurewebsites.net">NBTP</a>
-                <a class="dropdown-item" href="https://pecan.stjude.cloud">Pecan</a>
-                <a class="dropdown-item" href="http://sicklecelldev.azurewebsites.net">Sickle Cell</a>
-              </div>
+              <div class="dropdown-menu dropdown-menu-right dropdown-menu-main" aria-labelledby="nav-item-icon-waffle" id="dropdown-main">
+                <div class="d-flex flex-column flex-lg-row">
+                  <div class="nav-flex-fill">
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/data"><img alt="data-icon" class="icon-nav" src="../../dist/images/icon-nav-data.png">Data Browser</a>
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/tools"><img alt="tools-icon" class="icon-nav" src="../../dist/images/icon-nav-tools.png">Analysis Tools</a>
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/visualizations"><img alt="visualizations-icon" class="icon-nav" src="../../dist/images/icon-nav-visualization.png">Curated Visualizations</a>
+                  </div>
+                  <div class="nav-flex-fill">
+                    <a class="dropdown-column-header dropdown-item" href="">Apps</a>
+                    <a class="dropdown-item" href="https://pecan.stjude.cloud">PeCan</a>
+                    <a class="dropdown-item" href="https://pecan.stjude.cloud">Platform</a>
+                    <a class="dropdown-column-header dropdown-item" href="">Studies</a>
+                  </div>
+                  <div class="nav-flex-fill">
+                    <p class="dropdown-column-header">Help</p>
+                    <a class="dropdown-item" href="https://www.stjude.cloud/docs/" target="_blank">User Guides</a>
```

Comment:
If this is truly intended to open in a new tab, then add `rel="noopener noreferrer"`.